api/models/Detainees.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  attributes: {
+    person_id: {
+      type: 'string',
+      required: true,
+      unique: true,
+      primaryKey: true
+    },
+    cid_id: {
+      type: "integer"
+    },
+    gender: {
+      type: "string",
+      enum: ["male", "female"]
+    },
+    nationality: {
+      type: 'string'
+    },
+    centre: {
```

Comment:
Yes, I don't know why I didn't do that. I think I might have been distracted by `Centres.update`, which takes a centre name, and not a centre model, obviously, because that would be idiotic.